queries/java.scm

```diff
@@ -326,3 +326,32 @@
   body: (_) @value
   (#not-type? @value block)
 ) @_.domain
+
+;;!! public Map<int, int> foo;
+;;!         ^^^^^^^^^^^^^
+;;!  -------------------------
+(field_declaration
+  type: (_) @type
+) @_.domain
+
+;;!! class MyClass { }
+;;!                 ^
+(class_body
+  .
+  "{" @type.iteration.start.endOf
+  "}" @type.iteration.end.startOf
+  .
+)
+
+;;!! public Map<int, int> foo;
+;;!             ^^^  ^^^
+(integral_type) @type
```

Comment:
What is `integral_type` generally?

Reply:
Not what we want :p
It apparently only consists of the basic types. I have updated the pattern now.

Reply:
ok cool yeah that's what I had assumed based on its name 😅